lib/stripe/apple_pay_domain.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
+# Domains registered for Apple Pay on the Web
```

Comment:
This comment is more appropriate annotating the class (rather than `module Stripe`).

Reply:
bah, okay 😄